Resources/views/admin/fields/file-link-multiple.blade.php

```diff
@@ -1,54 +1,21 @@
 <script src="{{ Module::asset('dashboard:vendor/jquery-ui/jquery-ui.min.js') }}"></script>
```

Comment:
Shall i squash new pushes to this commit or add new ones?
Things like removing ``asgad.css`` from config and moving asset blocks from ``config.php`` to ``assets.php``